scripts/drone/events/release.star

```diff
@@ -462,9 +462,6 @@ def release_pipelines(ver_mode='release', trigger=None):
                 ]
             },
             'ref': ['refs/tags/v*',],
-            'repo': {
-                'exclude': ['grafana/grafana'],
-            },
```

Comment:
Hmmm. This is probably okay. Habit is sufficiently in place to not make this a problem.

Reply:
Yes! No problem with that, upon tagging, release builds are gonna run and put things into buckets. All good.